app/presenters/schools/on_boarding/school_profile_presenter.rb

```diff
@@ -8,6 +8,10 @@ def initialize(school_profile)
         @school = school_profile.bookings_school
       end
 
+      def ==(other)
+        other.instance_variable_get(:@school_profile) == @school_profile
```

Comment:
I think it would be better to expose the presenters profile with an `attr_reader` rather than bypassing scope and digging into the presenters internals